opendm/config.py

```diff
@@ -17,28 +31,34 @@ def __call__(self, parser, namespace, values, option_string=None):
 
 def config():
     parser.add_argument('--images', '-i',
-                        metavar='<string>',
+                        metavar='<path>',
                         help='Path to input images'),
 
     parser.add_argument('--project-path',
-                        metavar='<string>',
-                        help='Path to the project to process')
+                        metavar='<path>',
+                        help='Path to the project folder',
+                        default=defaultSettings['project_path'])
+
+    parser.add_argument('name',
```

Comment:
Sorry I should have updated the info but I ran out of time last week. Hopefully the updated comment above will help clear things up.